### ISSUE TITLE
Custom RSS import list

### ISSUE THREAD

**Issue by DJalup** [Readarr/Readarr#1845]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe

Currently there seems to be no direct way to import from RSS lists that are not listed as pre-configured.

### Describe the solution you'd like

To add the option for custom RSS feeds, This has already been done on Radarr
https://github.com/Radarr/Radarr/tree/627ab64fd023269c8bedece61e529329600a3419/src/NzbDrone.Core/ImportLists/RSSImport

### Describe alternatives you've considered

I tried adding the custom RSS feed to both Jackett and Prowlarr to go around this issue but it does not work.

### Anything else?

![customrss](https://user-images.githubusercontent.com/1342305/190216644-2162b478-3d47-4c1a-904d-e330bf96ad4e.png)

[AB#3735](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/3735)

**Comment by bakerboy448** [comment]
> I tried adding the custom RSS feed to both Jackett and Prowlarr to go around this issue but it does not work.

Neither of those apps have anything to do with book/author lists

**Comment by DJalup** [comment]
> > I tried adding the custom RSS feed to both Jackett and Prowlarr to go around this issue but it does not work.
> 
> Neither of those apps have anything to do with book/author lists

Yes, I am aware of that however trying to add the rss link in as an indexer so it would only search for the stuff that was in the rss feed was my attempt at a work-around.

**Comment by swiftsword94** [comment]
I would also like to see this included. What needs to be done to implement this change?
